ClosedXML/Excel/Cells/XLCell.cs

```diff
@@ -305,6 +305,8 @@ private string DeduceCellValueByParsing(string value, XLStyleValue style)
                 this.Style.SetIncludeQuotePrefix();
             }
             else if (!string.Equals(value.Trim(), "NaN", StringComparison.OrdinalIgnoreCase) &&
+                     !string.Equals(value.Trim(), "Infinity", StringComparison.OrdinalIgnoreCase) &&
```

Comment:
I think the list of 'disallowed' strings for parsing to `Double` should stored in a local static array.

Reply:
Very good point. I was thinking the same thing and was going to bring it up. I'll get on that.